<!DOCTYPE html>
<html lang="en-us">
  <head>
    
    <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="generator" content="Hugo 0.79.0 with theme Tranquilpeak 0.4.8-BETA">
<meta name="author" content="Columbia Data Science Institute (DSI) Scholars Program">
<meta name="keywords" content="">
<meta name="description" content="The hippocampus neurogenic niche (HNN) generates new neurons in mammals, but it is unclear if this is happening in humans. Adult hippocampal neurogenesis is necessary to maintain intact cognitive and emotional functions regulated by the hippocampus. Markers of immaturity have been detected in neuronal cells of the HNN but it is still unclear if they represent adult-born neurons, or neuronal cells that have maintained their immaturity since birth. We found that the number of neural progenitor cells (NPCs) and immature neurons was stable throughout the eighth decade of life in normal aging (NA) subjects, but angiogenesis and neuroplasticity were decreased in older people. Other groups have supported our findings, while some could not detect immature neuronal cells in human hippocampus. Moreover, in aging mice, more NPCs differentiate into glia rather than neurons, compared to younger animals, but we do not know if this happens in humans. Adult neurogenesis is lower in Alzheimer’s Disease (AD) and it is unknown if this is because more NPCs differentiate into glia or through other mechanisms. These gaps in knowledge warrant the use of new technologies to investigate cellular lineages in the human HNN, and molecular regulators of NPCs proliferation, cell fate, differentiation, maturation and survival. This project aims to identify differentially expressed proteins (DEPs) and genes (DEGs) in the human HNN, at the regional and single cell level, comparing NA and AD. We will apply our pipeline using high resolution mass spectrometry for proteomics analysis, and single nuclei (sn) RNA and ATAC (Assay for Transposase-Accessible Chromatin) sequencing (seq), to identify gene expression and epigenetic changes. In slide-mounted hippocampus tissue, we will apply Visium (10X Genomics) and our custom-made spatial transcriptomic technology for anatomical co-mapping of cell-type specific mRNAs and proteins (DBiT-seq). Novel computational approaches will identify neurogenesis regulators in the human HNN that can be tested in cellular or animal models. Findings obtained with these “OMICS” approaches will be validated using HighPlex RNAscope® (ADCBio) and immunofluorescence, and qPCR, Western blots, and ELISA assays, to visualize and quantify DEP and DEG expression at the single cell and regional level. Our rigorous brain collection methods assure tissue quality, uniform processing, use of toxicology and neuropathology, and strict clinical inclusion/exclusion criteria. Groups include: NA subjects (N=100), Braak stage 0-1, age 14-99 yrs., 40 of which (60 years of age and older) are matched (by age, sex and postmortem interval between death and brain collection) with 40 AD cases (from the Columbia Taub Institute collection), Braak stage 1 through 4. Aims: 1. Identify HNN DEPs associated with NA and AD. 2. Identify DEGs in immature and mature neuronal and glial cell populations of the DG in NA and AD subjects, using sn-RNA and sn-ATAC-seq (10X Genomics). 3. Determine the anatomical localization of cell expressing DEGs and DEPs associated with NA and AD, using Visium and DBiT-seq. 4. Test correlations between DEPs and DEGs, and numbers of NPCs and immature neurons and glia in NA and AD.">


<meta property="og:description" content="The hippocampus neurogenic niche (HNN) generates new neurons in mammals, but it is unclear if this is happening in humans. Adult hippocampal neurogenesis is necessary to maintain intact cognitive and emotional functions regulated by the hippocampus. Markers of immaturity have been detected in neuronal cells of the HNN but it is still unclear if they represent adult-born neurons, or neuronal cells that have maintained their immaturity since birth. We found that the number of neural progenitor cells (NPCs) and immature neurons was stable throughout the eighth decade of life in normal aging (NA) subjects, but angiogenesis and neuroplasticity were decreased in older people. Other groups have supported our findings, while some could not detect immature neuronal cells in human hippocampus. Moreover, in aging mice, more NPCs differentiate into glia rather than neurons, compared to younger animals, but we do not know if this happens in humans. Adult neurogenesis is lower in Alzheimer’s Disease (AD) and it is unknown if this is because more NPCs differentiate into glia or through other mechanisms. These gaps in knowledge warrant the use of new technologies to investigate cellular lineages in the human HNN, and molecular regulators of NPCs proliferation, cell fate, differentiation, maturation and survival. This project aims to identify differentially expressed proteins (DEPs) and genes (DEGs) in the human HNN, at the regional and single cell level, comparing NA and AD. We will apply our pipeline using high resolution mass spectrometry for proteomics analysis, and single nuclei (sn) RNA and ATAC (Assay for Transposase-Accessible Chromatin) sequencing (seq), to identify gene expression and epigenetic changes. In slide-mounted hippocampus tissue, we will apply Visium (10X Genomics) and our custom-made spatial transcriptomic technology for anatomical co-mapping of cell-type specific mRNAs and proteins (DBiT-seq). Novel computational approaches will identify neurogenesis regulators in the human HNN that can be tested in cellular or animal models. Findings obtained with these “OMICS” approaches will be validated using HighPlex RNAscope® (ADCBio) and immunofluorescence, and qPCR, Western blots, and ELISA assays, to visualize and quantify DEP and DEG expression at the single cell and regional level. Our rigorous brain collection methods assure tissue quality, uniform processing, use of toxicology and neuropathology, and strict clinical inclusion/exclusion criteria. Groups include: NA subjects (N=100), Braak stage 0-1, age 14-99 yrs., 40 of which (60 years of age and older) are matched (by age, sex and postmortem interval between death and brain collection) with 40 AD cases (from the Columbia Taub Institute collection), Braak stage 1 through 4. Aims: 1. Identify HNN DEPs associated with NA and AD. 2. Identify DEGs in immature and mature neuronal and glial cell populations of the DG in NA and AD subjects, using sn-RNA and sn-ATAC-seq (10X Genomics). 3. Determine the anatomical localization of cell expressing DEGs and DEPs associated with NA and AD, using Visium and DBiT-seq. 4. Test correlations between DEPs and DEGs, and numbers of NPCs and immature neurons and glia in NA and AD.">
<meta property="og:type" content="article">
<meta property="og:title" content="Multiomic spatial and single cell technologies in human brain disorders">
<meta name="twitter:title" content="Multiomic spatial and single cell technologies in human brain disorders">
<meta property="og:url" content="/2023/02/project-multiomic-spatial-and-single-cell-technologies-in-human-brain-disorders/">
<meta property="twitter:url" content="/2023/02/project-multiomic-spatial-and-single-cell-technologies-in-human-brain-disorders/">
<meta property="og:site_name" content="Columbia DSI Scholars">
<meta property="og:description" content="The hippocampus neurogenic niche (HNN) generates new neurons in mammals, but it is unclear if this is happening in humans. Adult hippocampal neurogenesis is necessary to maintain intact cognitive and emotional functions regulated by the hippocampus. Markers of immaturity have been detected in neuronal cells of the HNN but it is still unclear if they represent adult-born neurons, or neuronal cells that have maintained their immaturity since birth. We found that the number of neural progenitor cells (NPCs) and immature neurons was stable throughout the eighth decade of life in normal aging (NA) subjects, but angiogenesis and neuroplasticity were decreased in older people. Other groups have supported our findings, while some could not detect immature neuronal cells in human hippocampus. Moreover, in aging mice, more NPCs differentiate into glia rather than neurons, compared to younger animals, but we do not know if this happens in humans. Adult neurogenesis is lower in Alzheimer’s Disease (AD) and it is unknown if this is because more NPCs differentiate into glia or through other mechanisms. These gaps in knowledge warrant the use of new technologies to investigate cellular lineages in the human HNN, and molecular regulators of NPCs proliferation, cell fate, differentiation, maturation and survival. This project aims to identify differentially expressed proteins (DEPs) and genes (DEGs) in the human HNN, at the regional and single cell level, comparing NA and AD. We will apply our pipeline using high resolution mass spectrometry for proteomics analysis, and single nuclei (sn) RNA and ATAC (Assay for Transposase-Accessible Chromatin) sequencing (seq), to identify gene expression and epigenetic changes. In slide-mounted hippocampus tissue, we will apply Visium (10X Genomics) and our custom-made spatial transcriptomic technology for anatomical co-mapping of cell-type specific mRNAs and proteins (DBiT-seq). Novel computational approaches will identify neurogenesis regulators in the human HNN that can be tested in cellular or animal models. Findings obtained with these “OMICS” approaches will be validated using HighPlex RNAscope® (ADCBio) and immunofluorescence, and qPCR, Western blots, and ELISA assays, to visualize and quantify DEP and DEG expression at the single cell and regional level. Our rigorous brain collection methods assure tissue quality, uniform processing, use of toxicology and neuropathology, and strict clinical inclusion/exclusion criteria. Groups include: NA subjects (N=100), Braak stage 0-1, age 14-99 yrs., 40 of which (60 years of age and older) are matched (by age, sex and postmortem interval between death and brain collection) with 40 AD cases (from the Columbia Taub Institute collection), Braak stage 1 through 4. Aims: 1. Identify HNN DEPs associated with NA and AD. 2. Identify DEGs in immature and mature neuronal and glial cell populations of the DG in NA and AD subjects, using sn-RNA and sn-ATAC-seq (10X Genomics). 3. Determine the anatomical localization of cell expressing DEGs and DEPs associated with NA and AD, using Visium and DBiT-seq. 4. Test correlations between DEPs and DEGs, and numbers of NPCs and immature neurons and glia in NA and AD.">
<meta name="twitter:description" content="The hippocampus neurogenic niche (HNN) generates new neurons in mammals, but it is unclear if this is happening in humans. Adult hippocampal neurogenesis is necessary to maintain intact cognitive and emotional functions regulated by the hippocampus. Markers of immaturity have been detected in neuronal cells of the HNN but it is still unclear if they represent adult-born neurons, or neuronal cells that have maintained their immaturity since birth. We found that the number of neural progenitor cells (NPCs) and immature neurons was stable throughout the eighth decade of life in normal aging (NA) subjects, but angiogenesis and neuroplasticity were decreased in older people. Other groups have supported our findings, while some could not detect immature neuronal cells in human hippocampus. Moreover, in aging mice, more NPCs differentiate into glia rather than neurons, compared to younger animals, but we do not know if this happens in humans. Adult neurogenesis is lower in Alzheimer’s Disease (AD) and it is unknown if this is because more NPCs differentiate into glia or through other mechanisms. These gaps in knowledge warrant the use of new technologies to investigate cellular lineages in the human HNN, and molecular regulators of NPCs proliferation, cell fate, differentiation, maturation and survival. This project aims to identify differentially expressed proteins (DEPs) and genes (DEGs) in the human HNN, at the regional and single cell level, comparing NA and AD. We will apply our pipeline using high resolution mass spectrometry for proteomics analysis, and single nuclei (sn) RNA and ATAC (Assay for Transposase-Accessible Chromatin) sequencing (seq), to identify gene expression and epigenetic changes. In slide-mounted hippocampus tissue, we will apply Visium (10X Genomics) and our custom-made spatial transcriptomic technology for anatomical co-mapping of cell-type specific mRNAs and proteins (DBiT-seq). Novel computational approaches will identify neurogenesis regulators in the human HNN that can be tested in cellular or animal models. Findings obtained with these “OMICS” approaches will be validated using HighPlex RNAscope® (ADCBio) and immunofluorescence, and qPCR, Western blots, and ELISA assays, to visualize and quantify DEP and DEG expression at the single cell and regional level. Our rigorous brain collection methods assure tissue quality, uniform processing, use of toxicology and neuropathology, and strict clinical inclusion/exclusion criteria. Groups include: NA subjects (N=100), Braak stage 0-1, age 14-99 yrs., 40 of which (60 years of age and older) are matched (by age, sex and postmortem interval between death and brain collection) with 40 AD cases (from the Columbia Taub Institute collection), Braak stage 1 through 4. Aims: 1. Identify HNN DEPs associated with NA and AD. 2. Identify DEGs in immature and mature neuronal and glial cell populations of the DG in NA and AD subjects, using sn-RNA and sn-ATAC-seq (10X Genomics). 3. Determine the anatomical localization of cell expressing DEGs and DEPs associated with NA and AD, using Visium and DBiT-seq. 4. Test correlations between DEPs and DEGs, and numbers of NPCs and immature neurons and glia in NA and AD.">
<meta property="og:locale" content="en-us">

  
    <meta property="article:published_time" content="2023-02-22T00:00:00">
  
  
    <meta property="article:modified_time" content="2023-02-22T00:00:00">
  
  
  
    
      <meta property="article:section" content="Open  2023">
    
  
  
    
      <meta property="article:tag" content="2023">
    
  


<meta name="twitter:card" content="summary">

  <meta name="twitter:site" content="@dsi_columbia">


  <meta name="twitter:creator" content="@dsi_columbia">






  <meta property="og:image" content="/Users/ipekensari/Documents/GitHub/site-source/static/img/construction.png">
  <meta property="twitter:image" content="/Users/ipekensari/Documents/GitHub/site-source/static/img/construction.png">





  <meta property="og:image" content="http://res.cloudinary.com/tz33cu/image/upload/v1516632873/DSI-scholars/Data-Science-Avatar_jnwebr.png">
  <meta property="twitter:image" content="http://res.cloudinary.com/tz33cu/image/upload/v1516632873/DSI-scholars/Data-Science-Avatar_jnwebr.png">


    <title>Multiomic spatial and single cell technologies in human brain disorders</title>

    <link rel="icon" href="http://res.cloudinary.com/tz33cu/image/upload/c_scale,w_150/v1516739017/DSI-scholars/Data-Science-Avatar_qkn5fd.png">
    

    

    <link rel="canonical" href="../../../2023/02/project-multiomic-spatial-and-single-cell-technologies-in-human-brain-disorders/">

    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha256-eZrrJcwDc/3uDhsdt61sL2oOBY362qM3lon1gyExkL0=" crossorigin="anonymous" />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/2.1.4/jquery.fancybox.min.css" integrity="sha256-vuXZ9LGmmwtjqFX1F+EKin1ThZMub58gKULUyf0qECk=" crossorigin="anonymous" />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/2.1.4/helpers/jquery.fancybox-thumbs.min.css" integrity="sha256-SEa4XYAHihTcEP1f5gARTB2K26Uk8PsndQYHQC1f4jU=" crossorigin="anonymous" />
    
    
    <link rel="stylesheet" href="../../../css/style-twzjdbqhmnnacqs0pwwdzcdbt8yhv8giawvjqjmyfoqnvazl0dalmnhdkvp7.min.css" />
    
    

    
      
    
    
  </head>

  <body>
    <div id="blog">
      <header id="header" data-behavior="5">
  <i id="btn-open-sidebar" class="fa fa-lg fa-bars"></i>
  <div class="header-title">
    <a class="header-title-link" href="../../../">Columbia DSI Scholars</a>
  </div>
  
    
      <a class="header-right-picture "
         href="http://datascience.columbia.edu/">
    
    
    
      
        <img class="header-picture" src="http://res.cloudinary.com/tz33cu/image/upload/v1516632873/DSI-scholars/Data-Science-Avatar_jnwebr.png" alt="Author&#39;s picture" />
      
    
    </a>
  
</header>

      <nav id="sidebar" data-behavior="5">
  <div class="sidebar-container">
    
      <div class="sidebar-profile">
        <a href="../../../#about">
          <img class="sidebar-profile-picture" src="http://res.cloudinary.com/tz33cu/image/upload/v1516632873/DSI-scholars/Data-Science-Avatar_jnwebr.png" alt="Author&#39;s picture" />
        </a>
        <h4 class="sidebar-profile-name">Columbia Data Science Institute (DSI) Scholars Program</h4>
        
          <h5 class="sidebar-profile-bio">The DSI Scholars Program is to engage and support undergraduate and master students in participating data science related research with Columbia faculty. The program’s unique enrichment activities will foster a learning and collaborative community in data science at Columbia.</h5>
        
      </div>
    
    <ul class="sidebar-buttons">
      
  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="../../../">
    
      <i class="sidebar-button-icon fa fa-lg fa-home"></i>
      
      <span class="sidebar-button-desc">Home</span>
    </a>
  </li>

  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="../../../static/student_faq">
    
      <i class="sidebar-button-icon fa fa-lg fa-question-circle"></i>
      
      <span class="sidebar-button-desc">FAQ</span>
    </a>
  </li>

  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="../../../categories">
    
      <i class="sidebar-button-icon fa fa-lg fa-university"></i>
      
      <span class="sidebar-button-desc">Categories</span>
    </a>
  </li>

  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="../../../tags">
    
      <i class="sidebar-button-icon fa fa-lg fa-tags"></i>
      
      <span class="sidebar-button-desc">Tags</span>
    </a>
  </li>

  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="../../../#about">
    
      <i class="sidebar-button-icon fa fa-lg fa-info"></i>
      
      <span class="sidebar-button-desc">About</span>
    </a>
  </li>


    </ul>
    <ul class="sidebar-buttons">
      
  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="https://forms.gle/NE6MEprpTDxvxkMp6">
    
      <i class="sidebar-button-icon fa fa-lg fa-mortar-board"></i>
      
      <span class="sidebar-button-desc">Apply Now</span>
    </a>
  </li>

  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="../../../page/contact-info">
    
      <i class="sidebar-button-icon fa fa-lg fa-paper-plane"></i>
      
      <span class="sidebar-button-desc">Contact Info</span>
    </a>
  </li>

  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="https://github.com/CU-DSI-Scholars">
    
      <i class="sidebar-button-icon fa fa-lg fa-github"></i>
      
      <span class="sidebar-button-desc">GitHub</span>
    </a>
  </li>

  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="https://twitter.com/DSI_Columbia">
    
      <i class="sidebar-button-icon fa fa-lg fa-twitter"></i>
      
      <span class="sidebar-button-desc">Twitter</span>
    </a>
  </li>


    </ul>
    <ul class="sidebar-buttons">
      

    </ul>
  </div>
</nav>

      

      <div id="main" data-behavior="5"
        class="
               hasCoverMetaIn
               ">
        <article class="post" itemscope itemType="http://schema.org/BlogPosting">
          
          
            <div class="post-header main-content-wrap text-left">
  
    <h1 class="post-title" itemprop="headline">
      Multiomic spatial and single cell technologies in human brain disorders
    </h1>
  
  
  <div class="postShorten-meta post-meta">
    
      <time itemprop="datePublished" datetime="2023-02-22T00:00:00Z">
        
  February 22, 2023

      </time>
    
    
  
  
    <span>in</span>
    
      <a class="category-link" href="../../../categories/open-2023">Open  2023</a>
    
  

  </div>

</div>
          
          <div class="post-content markdown" itemprop="articleBody">
            <div class="main-content-wrap">
              <p>The hippocampus neurogenic niche (HNN) generates new neurons in mammals, but it is unclear if this is happening in humans. Adult hippocampal neurogenesis is necessary to maintain intact cognitive and emotional functions regulated by the hippocampus. Markers of immaturity have been detected in neuronal cells of the HNN but it is still unclear if they represent adult-born neurons, or neuronal cells that have maintained their immaturity since birth. We found that the number of neural progenitor cells (NPCs) and immature neurons was stable throughout the eighth decade of life in normal aging (NA) subjects, but angiogenesis and neuroplasticity were decreased in older people. Other groups have supported our findings, while some could not detect immature neuronal cells in human hippocampus. Moreover, in aging mice, more NPCs differentiate into glia rather than neurons, compared to younger animals, but we do not know if this happens in humans. Adult neurogenesis is lower in Alzheimer’s Disease (AD) and it is unknown if this is because more NPCs differentiate into glia or through other mechanisms. These gaps in knowledge warrant the use of new technologies to investigate cellular lineages in the human HNN, and molecular regulators of NPCs proliferation, cell fate, differentiation, maturation and survival. This project aims to identify differentially expressed proteins (DEPs) and genes (DEGs) in the human HNN, at the regional and single cell level, comparing NA and AD. We will apply our pipeline using high resolution mass spectrometry for proteomics analysis, and single nuclei (sn) RNA and ATAC (Assay for Transposase-Accessible Chromatin) sequencing (seq), to identify gene expression and epigenetic changes. In slide-mounted hippocampus tissue, we will apply Visium (10X Genomics) and our custom-made spatial transcriptomic technology for anatomical co-mapping of cell-type specific mRNAs and proteins (DBiT-seq). Novel computational approaches will identify neurogenesis regulators in the human HNN that can be tested in cellular or animal models. Findings obtained with these “OMICS” approaches will be validated using HighPlex RNAscope® (ADCBio) and immunofluorescence, and qPCR, Western blots, and ELISA assays, to visualize and quantify DEP and DEG expression at the single cell and regional level. Our rigorous brain collection methods assure tissue quality, uniform processing, use of toxicology and neuropathology, and strict clinical inclusion/exclusion criteria. Groups include: NA subjects (N=100), Braak stage 0-1, age 14-99 yrs., 40 of which (60 years of age and older) are matched (by age, sex and postmortem interval between death and brain collection) with 40 AD cases (from the Columbia Taub Institute collection), Braak stage 1 through 4. Aims: 1. Identify HNN DEPs associated with NA and AD. 2. Identify DEGs in immature and mature neuronal and glial cell populations of the DG in NA and AD subjects, using sn-RNA and sn-ATAC-seq (10X Genomics). 3. Determine the anatomical localization of cell expressing DEGs and DEPs associated with NA and AD, using Visium and DBiT-seq. 4. Test correlations between DEPs and DEGs, and numbers of NPCs and immature neurons and glia in NA and AD.</p>
<div class="alert success ">
  <p>This project is eligible for a stipend, with matching funds
from the faculty advisor and the Data Science Institute. This is not a guarantee of
payment, and the total amount is subject to available funding.</p>
</div>
<h2 id="faculty-advisor">Faculty Advisor</h2>
<ul>
<li>Professor: <a href="https://www.columbiapsychiatry.org/profile/maura-boldrini-md#research">Maura, Boldrini</a></li>
<li>Center/Lab: Psychiatry/Brain QUANT</li>
<li>Location: CUMC, Pardes Building, 3911B</li>
<li>In her laboratory, Dr. Boldrini is studying brain circuits involved in depression, suicide, aging  and other neurodegenerative conditions. She is studying brain circuits at the cellular and molecular level, and she is interested in understanding how the mind and the brain interact generating symptoms that make people suffer. The Brain QUANT Institute’s mission is to apply cutting edge quantitative neuroscience methods to interrogate, at a cellular and anatomical level, the genomics and molecular basis of brain biology and pathology. The ultimate goal is to identify new treatment targets and inform precision medicine approaches to brain disorders.</li>
</ul>
<h2 id="project-timeline">Project Timeline</h2>
<ul>
<li>Earliest starting date: 4/1/2023</li>
<li>End date:</li>
<li>Number of hours per week of research expected during Spring-Summer 2023: ~10</li>
</ul>
<h2 id="candidate-requirements">Candidate requirements</h2>
<ul>
<li>Skill sets: fluency in R/Python, Seurat</li>
<li>Student eligibility: <del>freshman</del>, <del>sophomore</del>, <del>junior</del>, <del>senior</del>, master&rsquo;s</li>
<li>International students on F1 or J1 visa: <strong>eligible</strong></li>
<li>Academic Credit Possible: Yes</li>
<li>Additional comments: Interest in genomics, multiomics, human brain</li>
</ul>
              
            </div>
          </div>
          <div id="post-footer" class="post-footer main-content-wrap">
            
              
                
                
                  <div class="post-footer-tags">
                    <span class="text-color-light text-small">TAGGED IN</span><br/>
                    
  <a class="tag tag--primary tag--small" href="../../../tags/2023/">2023</a>

                  </div>
                
              
            
            <div class="post-actions-wrap">
  
      <nav >
        <ul class="post-actions post-action-nav">
          
            <li class="post-action">
              
                <a class="post-action-btn btn btn--default tooltip--top" href="../../../2023/02/project-models-of-empirical-phenomena-in-deep-learning/" data-tooltip="Models of empirical phenomena in deep learning">
              
                  <i class="fa fa-angle-left"></i>
                  <span class="hide-xs hide-sm text-small icon-ml">NEXT</span>
                </a>
            </li>
            <li class="post-action">
              
                <a class="post-action-btn btn btn--default tooltip--top" href="../../../2023/02/project-opportunistic-weather-sensing-via-ml-and-data-analysis-using-link-measurements-from-a-nyc-wide-wireless-network/" data-tooltip="Opportunistic Weather Sensing via ML and data analysis, using Link Measurements from a NYC-Wide Wireless Network">
              
                  <span class="hide-xs hide-sm text-small icon-mr">PREVIOUS</span>
                  <i class="fa fa-angle-right"></i>
                </a>
            </li>
          
        </ul>
      </nav>
    <ul class="post-actions post-action-share" >
      
        <li class="post-action hide-lg hide-md hide-sm">
          <a class="post-action-btn btn btn--default btn-open-shareoptions" href="#btn-open-shareoptions">
            <i class="fa fa-share-alt"></i>
          </a>
        </li>
        
          <li class="post-action hide-xs">
            <a class="post-action-btn btn btn--default" target="new" href="https://www.facebook.com/sharer/sharer.php?u=/2023/02/project-multiomic-spatial-and-single-cell-technologies-in-human-brain-disorders/">
              <i class="fa fa-facebook-official"></i>
            </a>
          </li>
        
          <li class="post-action hide-xs">
            <a class="post-action-btn btn btn--default" target="new" href="https://twitter.com/intent/tweet?text=/2023/02/project-multiomic-spatial-and-single-cell-technologies-in-human-brain-disorders/">
              <i class="fa fa-twitter"></i>
            </a>
          </li>
        
          <li class="post-action hide-xs">
            <a class="post-action-btn btn btn--default" target="new" href="https://plus.google.com/share?url=../../../2023/02/project-multiomic-spatial-and-single-cell-technologies-in-human-brain-disorders/">
              <i class="fa fa-google-plus"></i>
            </a>
          </li>
        
      
      
      <li class="post-action">
        
          <a class="post-action-btn btn btn--default" href="#">
        
          <i class="fa fa-list"></i>
        </a>
      </li>
    </ul>
  
</div>

            
              
            
          </div>
        </article>
        <footer id="footer" class="main-content-wrap">
  <span class="copyrights">
    &copy; 2023 Columbia Data Science Institute (DSI) Scholars Program. All Rights Reserved
  </span>
</footer>

      </div>
      <div id="bottom-bar" class="post-bottom-bar" data-behavior="5">
        <div class="post-actions-wrap">
  
      <nav >
        <ul class="post-actions post-action-nav">
          
            <li class="post-action">
              
                <a class="post-action-btn btn btn--default tooltip--top" href="../../../2023/02/project-models-of-empirical-phenomena-in-deep-learning/" data-tooltip="Models of empirical phenomena in deep learning">
              
                  <i class="fa fa-angle-left"></i>
                  <span class="hide-xs hide-sm text-small icon-ml">NEXT</span>
                </a>
            </li>
            <li class="post-action">
              
                <a class="post-action-btn btn btn--default tooltip--top" href="../../../2023/02/project-opportunistic-weather-sensing-via-ml-and-data-analysis-using-link-measurements-from-a-nyc-wide-wireless-network/" data-tooltip="Opportunistic Weather Sensing via ML and data analysis, using Link Measurements from a NYC-Wide Wireless Network">
              
                  <span class="hide-xs hide-sm text-small icon-mr">PREVIOUS</span>
                  <i class="fa fa-angle-right"></i>
                </a>
            </li>
          
        </ul>
      </nav>
    <ul class="post-actions post-action-share" >
      
        <li class="post-action hide-lg hide-md hide-sm">
          <a class="post-action-btn btn btn--default btn-open-shareoptions" href="#btn-open-shareoptions">
            <i class="fa fa-share-alt"></i>
          </a>
        </li>
        
          <li class="post-action hide-xs">
            <a class="post-action-btn btn btn--default" target="new" href="https://www.facebook.com/sharer/sharer.php?u=/2023/02/project-multiomic-spatial-and-single-cell-technologies-in-human-brain-disorders/">
              <i class="fa fa-facebook-official"></i>
            </a>
          </li>
        
          <li class="post-action hide-xs">
            <a class="post-action-btn btn btn--default" target="new" href="https://twitter.com/intent/tweet?text=/2023/02/project-multiomic-spatial-and-single-cell-technologies-in-human-brain-disorders/">
              <i class="fa fa-twitter"></i>
            </a>
          </li>
        
          <li class="post-action hide-xs">
            <a class="post-action-btn btn btn--default" target="new" href="https://plus.google.com/share?url=../../../2023/02/project-multiomic-spatial-and-single-cell-technologies-in-human-brain-disorders/">
              <i class="fa fa-google-plus"></i>
            </a>
          </li>
        
      
      
      <li class="post-action">
        
          <a class="post-action-btn btn btn--default" href="#">
        
          <i class="fa fa-list"></i>
        </a>
      </li>
    </ul>
  
</div>

      </div>
      <div id="share-options-bar" class="share-options-bar" data-behavior="5">
  <i id="btn-close-shareoptions" class="fa fa-close"></i>
  <ul class="share-options">
    
      <li class="share-option">
        <a class="share-option-btn" target="new" href="https://www.facebook.com/sharer/sharer.php?u=%2F2023%2F02%2Fproject-multiomic-spatial-and-single-cell-technologies-in-human-brain-disorders%2F">
          <i class="fa fa-facebook-official"></i><span>Share on Facebook</span>
        </a>
      </li>
    
      <li class="share-option">
        <a class="share-option-btn" target="new" href="https://twitter.com/intent/tweet?text=%2F2023%2F02%2Fproject-multiomic-spatial-and-single-cell-technologies-in-human-brain-disorders%2F">
          <i class="fa fa-twitter"></i><span>Share on Twitter</span>
        </a>
      </li>
    
      <li class="share-option">
        <a class="share-option-btn" target="new" href="https://plus.google.com/share?url=%2F2023%2F02%2Fproject-multiomic-spatial-and-single-cell-technologies-in-human-brain-disorders%2F">
          <i class="fa fa-google-plus"></i><span>Share on Google&#43;</span>
        </a>
      </li>
    
  </ul>
</div>
<div id="share-options-mask" class="share-options-mask"></div>
    </div>
    
    <div id="about">
  <div id="about-card">
    <div id="about-btn-close">
      <i class="fa fa-remove"></i>
    </div>
    
      <img id="about-card-picture" src="http://res.cloudinary.com/tz33cu/image/upload/v1516632873/DSI-scholars/Data-Science-Avatar_jnwebr.png" alt="Author&#39;s picture" />
    
    <h4 id="about-card-name">Columbia Data Science Institute (DSI) Scholars Program</h4>
    
      <div id="about-card-bio">The DSI Scholars Program is to engage and support undergraduate and master students in participating data science related research with Columbia faculty. The program’s unique enrichment activities will foster a learning and collaborative community in data science at Columbia.</div>
    
    
      <div id="about-card-job">
        <i class="fa fa-briefcase"></i>
        <br/>
        Columbia University DSI
      </div>
    
    
      <div id="about-card-location">
        <i class="fa fa-map-marker"></i>
        <br/>
        New York, NY
      </div>
    
  </div>
</div>

    

    
  
    <div id="cover" style="background-image:url('https://github.com/CU-DSI-Scholars/site-source/blob/main/static/img/CU-bk2.png?raw=true');"></div>
  


    
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.4/jquery.min.js" integrity="sha256-BbhdlvQf/xTY9gja0Dq3HiwQF8LaCRTXxZKRutelT44=" crossorigin="anonymous"></script>

  <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js" integrity="sha256-/BfiIkHlHoVihZdc6TFuj7MmJ0TWcWsMXkeDFwhi0zw=" crossorigin="anonymous"></script>

<script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/2.1.7/js/jquery.fancybox.min.js" integrity="sha256-GEAnjcTqVP+vBp3SSc8bEDQqvWAZMiHyUSIorrWwH50=" crossorigin="anonymous"></script>


<script src="../../../js/script-pcw6v3xilnxydl1vddzazdverrnn9ctynvnxgwho987mfyqkuylcb1nlt.min.js"></script>


<script lang="javascript">
window.onload = updateMinWidth;
window.onresize = updateMinWidth;
document.getElementById("sidebar").addEventListener("transitionend", updateMinWidth);
function updateMinWidth() {
  var sidebar = document.getElementById("sidebar");
  var main = document.getElementById("main");
  main.style.minWidth = "";
  var w1 = getComputedStyle(main).getPropertyValue("min-width");
  var w2 = getComputedStyle(sidebar).getPropertyValue("width");
  var w3 = getComputedStyle(sidebar).getPropertyValue("left");
  main.style.minWidth = `calc(${w1} - ${w2} - ${w3})`;
}
</script>

<script>
$(document).ready(function() {
  hljs.configure({ classPrefix: '', useBR: false });
  $('pre.code-highlight > code, pre > code').each(function(i, block) {
    if (!$(this).hasClass('codeblock')) {
      $(this).addClass('codeblock');
    }
    hljs.highlightBlock(block);
  });
});
</script>


  
    
  




    
  </body>
</html>

